torch/testing/_internal/common_utils.py

```diff
@@ -1049,6 +1049,46 @@ def setUp(self):
 
         set_rng_seed(SEED)
 
+    def genSparseCSRTensor(self, size, nnz, device, dtype, index_dtype):
```

Comment:
Add a comment for this function

I suggest changing its signature to `(self, size, nnz, *, device, dtype, index_dtype)` for readability